### PR TITLE
Added jobs to check clang-format

### PR DESCRIPTION
**Pull request by vitalych**
Signed-off-by: Vitaly Chipounov <vitaly@cyberhaven.io>

### CODE REVIEW

File: jenkins/jobs-templates/generate_jobs.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    parser.add_argument('-t', '--templates-dir', required=True, help='Path to the templates directory')
+    args = parser.parse_args()
+
+    if not os.path.isdir(args.jobs_dir):
+        log.error('%s does not exist', args.jobs_dir)
+        exit(1)
+
+    if not os.path.isdir(args.templates_dir):
+        log.error('%s does not exist', args.templates_dir)
+        exit(1)
```
Comment (adrianherrera) [inline jenkins/jobs-templates/generate_jobs.py]: Does this work? Isn’t `exit` provided by the `sys` module?

I.e., `sys.exit(1)`

**Review by adrianherrera — CHANGES_REQUESTED** [review]
Double check that you are using exit correctly, AFAIK it should be `sys.exit`). Running `pylint` over the file should suffice

File: jenkins/jobs-templates/generate_jobs.py (hunk 1 above)
Comment (vitalych) [reply jenkins/jobs-templates/generate_jobs.py]: Yes, but apparently it should not be used, so I'll fix it. Pylint didn't complain.
https://stackoverflow.com/questions/19747371/python-exit-commands-why-so-many-and-when-should-each-be-used

**Comment by adrianherrera** [comment]
Oh wow, I didn’t know that. Learn something new every day...

**Review by adrianherrera — APPROVED** [review]

**Pull request merged**